app/views/constraints/edit.html.erb

```diff
@@ -56,8 +54,8 @@
       <% end %>
 
       <div class="form-group">
-        <%= form.label :old_constraints, "Add a local constraint", class: 'govuk-heading-s govuk-!-margin-top-6' %>
-        <%= form.text_area :old_constraints, class: "govuk-textarea", rows: "3", "aria-describedby": "constraints-hint", multiple: true, value: "" %>
+        <%= form.label :name, "Add a local constraint", class: 'govuk-heading-s govuk-!-margin-top-6' %>
+        <%= form.text_area :name, class: "govuk-textarea", rows: "3", "aria-describedby": "constraints-hint" %>
```

Comment:
I am wondering whether we should add send everything as `capitalize` or `titleize` here. For a example if someone adds `Flood zone 4` or `Flood Zone 4` or even `Flood zone 4  ` it should be stored it as `Flood zone 4`. I know that's an improvement to the current flow and maybe not part of this. What do you think?

Reply:
Added here - https://github.com/unboxed/bops/pull/1106/files#diff-56352b080a99ea83abab463a40b35fc4f34425f8e84ec8c1f7117e22d2783b62R21